### PR TITLE
Add test cases for emr_config, pypi/maven/npm emrs

### PR DESCRIPTION
**Pull request by ravsa**

### CODE REVIEW

File: tests/data_store/test_aws.py [hunk 1]
```diff
@@ -226,5 +226,76 @@ def test_run_flow(self, emr):
         assert status_code == 200
         job_flow_id = response.get('JobFlowId')
         assert job_flow_id
-        cluster_info = emr._emr.describe_cluster(ClusterId=job_flow_id)
-        assert cluster_info.get("Cluster", {}).get("Name") == "HPF_training"
+        job_status = emr.get_status(job_flow_id)
+        assert job_status.get('State') == 'WAITING'
+
+    def test_terminate_job(self, emr):
```
Comment (miteshvp) [inline tests/data_store/test_aws.py]: Missing doc-style defs

File: tests/data_store/test_aws.py [hunk 2]
```diff
@@ -226,5 +226,76 @@ def test_run_flow(self, emr):
...
+            'ResponseMetadata', {}).get('HTTPStatusCode')
+        assert status_code == 200
+        job_flow_id = response.get('JobFlowId')
+        job_status = emr.get_status(job_flow_id)
+        assert job_status.get('State') == 'WAITING'
+        emr.terminate_jobs(job_flow_id)
+        job_status = emr.get_status(job_flow_id)
+        assert job_status.get('State') == 'TERMINATED'
+
+    def test_terminate_jobs(self, emr):
```
Comment (miteshvp) [inline tests/data_store/test_aws.py]: Again, missing doc-style definitions

File: tests/deployments/emrs/test_emr_config.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+from rudra.deployments.emrs.emr_config import EMRConfig
+
+
+class TestEMRConfig:
```
Comment (miteshvp) [inline tests/deployments/emrs/test_emr_config.py]: Missing class and function definitions doc-style comments

File: tests/data_store/test_aws.py (hunk 1 above)
Comment (ravsa) [reply tests/data_store/test_aws.py]: Thanks, @miteshvp. Actually, I am not adding doc-styles for any of the tests.

**Pull request merged**